src/js/components/validateData/validateValues/ValidateValuesErrorReport.jsx

```diff
@@ -66,7 +66,7 @@ export default class ValidateValuesErrorReport extends React.Component {
 
             let description = item.error_description;
             if (item.error_name == 'rule_failed') {
-                description = item.rule_failed;
+                description = ['Rule ' + item.original_label + ': ' + item.rule_failed];
```

Comment:
As a side note, I don't think you want to put it into an array. You can just make it a string